sos/report/plugins/networkmanager.py

```diff
@@ -19,6 +20,10 @@ class NetworkManager(Plugin, RedHatPlugin, UbuntuPlugin):
     packages = ('NetworkManager', 'network-manager')
 
     def setup(self):
+        nm_pkg = self.policy.package_manager.pkg_by_name('NetworkManager')
+        if nm_pkg is None:
+            return
+
```

Comment:
This section is not required. The plugin will only come into use if the items in the `packages` tuple

Reply:
Understood. I removed the section.